perfkitbenchmarker/azure/__init__.py

```diff
@@ -11,3 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from perfkitbenchmarker.azure import util
+
+util.CheckAzureVersion()
```

Comment:
Sure I can do that - checking FLAGS.cloud won't be sufficient in the future, but it should work for now.